test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -37,12 +37,29 @@ call_log_manager::call_log_manager(const std::string &call_log_file)
 {
     std::ifstream file(call_log_file);
     if (file.fail()) {
-        std::cout << "File '" << call_log_file << "' either doesn't exist or is not accessible."
+        std::cerr << "File '" << call_log_file << "' either doesn't exist or is not accessible."
                   << std::endl;
         exit(1);
     }
 
-    _call_log = json::parse(file);
+    std::string contents(
+      (std::istreambuf_iterator<char>(file)), (std::istreambuf_iterator<char>()));
+
+    if (contents.empty()) {
+        std::cerr << "File'" << call_log_file << "' is empty." << std::endl;
+        exit(1);
+    }
+    /*
+     * Get rid of the end line and comma from the call_log if it exists, and prepend, append square
+     * brackets.
+     */
+    if (contents.back() == '\n')
```

Comment:
I would probably do a while just in case. Same for the comma.

Reply:
Updated in [76ee71a](https://github.com/wiredtiger/wiredtiger/pull/8186/commits/76ee71a2f16b8cedf531ca0fa2b1820284311c3c)